microvmConfigurations/guivm/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  nixpkgs,
+  microvm,
+  system,
+}:
+nixpkgs.lib.nixosSystem {
+  inherit system;
+
+  modules = [
+    # TODO: Enable only for development builds
+    ../../modules/development/authentication.nix
+    ../../modules/development/ssh.nix
+    ../../modules/development/packages.nix
+
+    microvm.nixosModules.microvm
+
+    ({pkgs, ...}: {
+      networking.hostName = "guivm";
+      # TODO: Maybe inherit state version
+      system.stateVersion = "22.11";
+
+      microvm.hypervisor = "qemu";
+
+      nixpkgs.overlays = [
+        (final: prev: {
+          qemu_kvm = prev.qemu_kvm.overrideAttrs (final: prev: {
+            patches = prev.patches ++ [./qemu-aarch-memory.patch];
+          });
+        })
+      ];
+
+      networking.enableIPv6 = false;
+      networking.interfaces.eth0.useDHCP = true;
+      networking.firewall.allowedTCPPorts = [22];
+
+      environment.systemPackages = with pkgs; [
+        weston
+      ];
+
+      hardware.nvidia.open = false;
+      hardware.nvidia.modesetting.enable = true;
+      nixpkgs.config.allowUnfree = true;
+      services.xserver.videoDrivers = ["nvidia"];
```

Comment:
I actually wouldn't like to enable the NVIDIA's closed source drivers by default